website/docs/docs/available-adapters.md

```diff
@@ -21,6 +21,14 @@ In addition to maintaining `dbt-core`, [Fishtown Analytics](https://github.com/f
 
 Adapters distributed with "core" are ready to use when you [install dbt](dbt-cli/installation). For "plugin" adapters, check each page for specific installation instructions.
 
+### Vendor Supported
+
+These adapter plugins are built and maintained by the same people who build and maintain the complementary data technology.
+
+| Adapter for  | Documentation |
+| ------------ | ------------- |
+| Materialize ([dbt-materialize](https://github.com/MaterializeInc/materialize/blob/main/misc/dbt-materialize))  | [Profile Setup](materialize-profile) |
```

Comment:
This is more of my own curiosity but is there a reason why this lives in a misc folder rather than a separate repo from the Materialize product repo?

Reply:
Keeping it in `misc` allows us to run the `dbt-adapter-tests` as part of our CI workflow!

Reply:
ah! That's good to know! Thanks @JLDLaughlin